packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1190,6 +1346,11 @@ export interface SpecWatchEntity {
   removedTargets?: TargetId[];
 }
 
+export type SpecClientState = {
```

Comment:
Can you add a "PORTING NOTE: Only used by web multi-tab tests." ?

Reply:
Added here and in a bunch of other places.